### PR TITLE
docs: update the claims section

### PR DESCRIPTION
**Pull request by madarche**
This is not the ultimate wording, but at least it clearly shows that `claims` does actually 2 kind of different things with a different syntax

### CODE REVIEW

**Review by panva — CHANGES_REQUESTED** [review]
@madarche thank you. Please make your change in defaults.js and then run ./docs/update-configuration.js

Also, use `-` for the list instead of `*`

**Comment by madarche** [comment]
@panva OK!

**Comment by madarche** [comment]
@panva I've leveraged `docs/update-configuration.js`. This is a neat tool!

File: docs/README.md [hunk 1]
```diff
@@ -1961,7 +1961,10 @@ async function audiences(ctx, sub, token, use) {
 
 ### claims
 
-Describes the claims that the OpenID Provider MAY be able to supply values for. It is also used to define which claims fall under what scope (configure `{ scopeName: ['claim', 'another-claim'] }`) as well as to expose additional claims that are available to RPs via the `claims` authorization parameter (configure as `{ claimName: null }`).   
+Defines the claims that the OpenID Provider MAY be able to supply values for.
```
Comment (panva) [inline docs/README.md:1964]:
```suggestion
Describes the claims that the OpenID Provider MAY be able to supply values for.   
```

File: lib/helpers/defaults.js [hunk 2]
```diff
@@ -485,11 +485,11 @@ function getDefaults() {
     /*
      * claims
      *
-     * description: Describes the claims that the OpenID Provider MAY be able to
-     *   supply values for. It is also used to define which claims fall under what scope (configure
-     *   `{ scopeName: ['claim', 'another-claim'] }`) as well as to expose additional claims that
-     *   are available to RPs via the `claims` authorization parameter (configure as
-     *   `{ claimName: null }`).
+     * description: Defines the claims that the OpenID Provider MAY be able to supply values for.
```
Comment (panva) [inline lib/helpers/defaults.js:488]:
```suggestion
     * description: Describes the claims that the OpenID Provider MAY be able to supply values for.
```

File: docs/README.md [hunk 3]
```diff
@@ -1961,7 +1961,10 @@ async function audiences(ctx, sub, token, use) {
 
 ### claims
 
-Describes the claims that the OpenID Provider MAY be able to supply values for. It is also used to define which claims fall under what scope (configure `{ scopeName: ['claim', 'another-claim'] }`) as well as to expose additional claims that are available to RPs via the `claims` authorization parameter (configure as `{ claimName: null }`).   
+Defines the claims that the OpenID Provider MAY be able to supply values for.   
+ It defines 2 different things:
```
Comment (panva) [inline docs/README.md:1965]:
```suggestion
 It is used to achieve two different things related to claims:
```

File: lib/helpers/defaults.js [hunk 4]
```diff
@@ -485,11 +485,11 @@ function getDefaults() {
...
+     *
+     * It defines 2 different things:
```
Comment (panva) [inline lib/helpers/defaults.js:490]:
```suggestion
     * It is used to achieve two different things related to claims:
```

**Pull request merged**